app/js/reducers/fetchOrdersReducer.js

```diff
@@ -1,5 +1,10 @@
+import { all } from 'redux-saga/effects';
+
 const initialState = {
-  orders: {},
+  orders: {
+    results: [],
```

Comment:
I know this was the way it was previously, but this might be good time to refactor to just:

`orders: []`

... and then update the components that map in this state to the new format.

Having the orders nested under "results" seems unnecessary, and is now inconsistent with the new:

`filteredOrders: []
`

Reply:
addressed in comments below